pkg/file/video/caption.go

```diff
@@ -98,17 +98,25 @@ func AssociateCaptions(ctx context.Context, captionPath string, txnMgr txn.Manag
 	captionPrefix := getCaptionPrefix(captionPath)
 	if err := txn.WithTxn(ctx, txnMgr, func(ctx context.Context) error {
 		var err error
-		f, er := fqb.FindByPath(ctx, captionPrefix+"*")
+		files, er := fqb.FindAllByPath(ctx, captionPrefix+"*")
 
 		if er != nil {
 			return fmt.Errorf("searching for scene %s: %w", captionPrefix, er)
 		}
 
-		if f != nil { // found related Scene
+		for _, f := range files {
+			// found some files
+			// filter out non video files
+			switch f.(type) {
+			case *file.VideoFile:
+				break
+			default:
+				continue
+			}
+
 			fileID := f.Base().ID
 			path := f.Base().Path
 
-			logger.Debugf("Matched captions to file %s", path)
```

Comment:
I'd leave this message in place. This message and the one above are here to assist in debugging. If I had remembered these messages were here, I might have been able to identify that this was the issue another user was having.

Reply:
Done, removed the last commit